### PR TITLE
Changes made for button

### PR DESCRIPTION
**Pull request by kisharnath**

### CODE REVIEW

**Review by manniL — CHANGES_REQUESTED** [review]
Thanks for the initial changes! Could we:

1) Stick to TailwindCSS?
2)  Still use the AppButton for the buttons? 🤔

**Comment by kisharnath** [comment]
I am unable to find the right class for this ,Should I make custom class in tailwind?
And (Still use the AppButton for the buttons? )  This means can AppButton be used as button am I right?

**Comment by manniL** [comment]
@kisharnath The idea that this button should work with less elements. If tailwind doesn't have the utilities for it, you can use it's [arbitrary syntax](https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values).
Default: image
On Hover:
image

**Comment by kisharnath** [comment]
Can you check now?

File: assets/css/tailwind.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
+
+.newButton{
```
Comment (manniL) [inline assets/css/tailwind.css:5]: As mentioned, please use Tailwinds Arbitrary syntax ☺️ 
If that is not possible at all, Vue's scoped styles are also fine, but then use `@apply` where possible

**Review by manniL — CHANGES_REQUESTED** [review]

**Comment by kisharnath** [comment]
Can you check out now ?

**Comment by manniL** [comment]
Hey @kisharnath! Thanks for your work so far. Unfortunately, the styles don't match the original. The border on the side is thinner than on the top and bottom. Also, when hovering there is no gradient anymore.
image

**Comment by kisharnath** [comment]
Please  check now

**Comment by manniL** [comment]
@kisharnath can you check my comments @ https://youtu.be/K-2r2rXHEdo?t=6528? ☺️

**Pull request closed without merging**